jans-cedarling/flask-sidecar/main/config.py

```
@@ -43,23 +43,22 @@ class BaseConfig:
-                f"Unable to read Cedarling bootstrap config from {CEDARLING_BOOTSTRAP_CONFIG_FILE}: {e}. "
-                "Falling back to environment variables"
-            )
+        raise RuntimeError("CEDARLING_BOOTSTRAP_CONFIG_FILE is not set")
+    try:
+        with open(CEDARLING_BOOTSTRAP_CONFIG_FILE, "r") as f:
+            CEDARLING_BOOTSTRAP_CONFIG = f.read()
+    except OSError as e:
+        logger.warning(
+            f"Unable to read Cedarling bootstrap config from {CEDARLING_BOOTSTRAP_CONFIG_FILE}: {e}. Exiting..."
+        )
+        raise RuntimeError("CEDARLING_BOOTSTRAP_CONFIG_FILE is not set")
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)